etcd.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: etcd
+  version: 3.5.6
+  epoch: 0
+  description: A highly-available key value store for shared configuration and service discovery.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - git
+      - bash
+      - go
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/coreos/etcd/archive/v${{package.version}}.tar.gz
+      expected-sha256: 0ef11c0cdc3cc693756a636d97d280ea63e222a7be38159fad0c8032ced0da42
+  - runs: |
+      bash -x ./build.sh
+      mkdir -p "${{targets.destdir}}"/var/lib/$pkgname
```

Comment:
`$pkgname` can change to the melange templated value `${{package.name}}` or just hard code `etcd`

Reply:
Fixed!

Reply:
I've definitely made this mistake before. We should add a check for this in linting.